include/cuco/detail/pair/traits.hpp

```diff
@@ -15,11 +15,11 @@
 
 #pragma once
 
+#include <cuda/std/tuple>
+#include <cuda/std/type_traits>
 #include <thrust/device_reference.h>
 #include <thrust/tuple.h>
 
-#include <cuda/std/tuple>
-#include <cuda/std/type_traits>
 #include <tuple>
```

Comment:
Not sure if we should group `cuda::std` facilities with `thrust` rather than STL.

Reply:
We are using a new convention in cudf https://github.com/rapidsai/cudf/pull/15063.

The idea is that thrust/cub/libcu++ all belongs to CCCL thus they are as one single header group.

Reply:
That makes alot of sense. I stand corrected.